### DISCUSSION TITLE
segnalazione

### DISCUSSION THREAD

**Discussion by francyoo17** [iisgiua/giuaschool#781] [Idee e proposte]
vorrei segnalare che utente registro-giuaschool con emai:registroelettronicogiuaschool@gmail.com) ruba gli account degli utenti del registro e delle console google

**Comment by trinko** [comment]
Sei sicuro di quanto affermi dovresti fare una segnlazione alla polizia postale fornendo tutte le informazioni necessarie per identificare tale persona.

Poi sarebbe opportuno rivedere le impostazioni di sicurezza del server che utilizzi: è attivo l'HTTPS? Inoltre dovresti assicurarti che tutti gli utenti abbiano la massima cura delle proprie credenziali e che siano consapevoli dei rischi della loro divulgazione.
